tools/php-cs/.styleci.yml

```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+#
+# Package `sllh/php-cs-fixer-styleci-bridge` is required to get it working.
+
+preset: symfony
+
+enabled:
+  - align_double_arrow
```

Comment:
I think the closer it stays from PSR-2 + Symfony cs rules, the less friction for everybody.